src/lib/pageCommandProperties.ts

```diff
@@ -57,20 +57,20 @@ export function examples(section: Section) {
 
 export function baseFlags(section: Section) {
   const fileType = section === 'Custom Pages' ? 'Markdown/HTML' : 'Markdown';
-  let items;
```

Comment:
It didn't like that `items` didn't have an explicit type.

Reply:
Rolling this change back and adding an ignore w/ an explanation that the   switch statement gives us our types.